arangod/VocBase/Methods/Indexes.cpp

```diff
@@ -497,14 +499,22 @@ Result Indexes::ensureIndex(LogicalCollection* collection, VPackSlice input,
         collection->getPhysical()->flushClusterIndexEstimates();
 
         // the cluster won't set a proper id value
-        std::string iid = tmp.slice().get(StaticStrings::IndexId).copyString();
-        VPackBuilder b;
-        b.openObject();
-        b.add(StaticStrings::IndexId,
-              VPackValue(collection->name() + TRI_INDEX_HANDLE_SEPARATOR_CHR +
-                         iid));
-        b.close();
-        output = VPackCollection::merge(tmp.slice(), b.slice(), false);
+        // we don't need analyzer revision here
+        output.openObject();
```

Comment:
Please ad an assert that output is empty (it should be, but just in case)

Reply:
Done